package javaStudy.ch8.ex;

public class Exercise8_4 {
	//  c. Exception이 첫번째 catch문에 있으면 다음 catch문을 정의할 필요가 없다.
}
